2020/10/27-00:11  - Iniciando o BATCH Start.sh
2020/10/27-00:11  - Aguardando 5s - Start
2020/10/27-00:11  - iniciando o programa
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    bot.data_manager.load_config()
  File "D:\Desenvolvimento\bot-discord\Resources\Data.py", line 24, in load_config
    config = self.bot.yaml.load(file)
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\main.py", line 341, in load
    constructor, parser = self.get_constructor_parser(stream)
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\main.py", line 397, in get_constructor_parser
    self.reader.stream = stream
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\reader.py", line 130, in stream
    self.determine_encoding()
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\reader.py", line 190, in determine_encoding
    self.update_raw()
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\reader.py", line 297, in update_raw
    data = self.stream.read(size)
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 266: character maps to <undefined>
1
2020/10/27-00:11  - Restart no programa
2020/10/27-00:11  - Aguardando 5s - Restart
2020/10/27-00:11  - iniciando o programa
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    bot.data_manager.load_config()
  File "D:\Desenvolvimento\bot-discord\Resources\Data.py", line 24, in load_config
    config = self.bot.yaml.load(file)
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\main.py", line 341, in load
    constructor, parser = self.get_constructor_parser(stream)
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\main.py", line 397, in get_constructor_parser
    self.reader.stream = stream
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\reader.py", line 130, in stream
    self.determine_encoding()
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\reader.py", line 190, in determine_encoding
    self.update_raw()
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\site-packages\ruamel\yaml\reader.py", line 297, in update_raw
    data = self.stream.read(size)
  File "C:\Users\Nadai\AppData\Local\Programs\Python\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 266: character maps to <undefined>
1
2020/10/27-00:11  - Restart no programa
2020/10/27-00:11  - Aguardando 5s - Restart
